src/string_tensor_unpack.hpp

```diff
@@ -5,19 +5,22 @@
 #pragma once
 
 #include <openvino/op/op.hpp>
+#include <openvino/op/string_tensor_unpack.hpp>
 
+// THIS CLASS IS DEPRECATED: This class is deprecated and will it's left here only for purpose of backward compatibility with old IRs.
+// Please use op::v15::StringTensorUnpack from official opset instead.
 // Unpack a string tensor representation regardless of the source format, which
 // can be an OpenVINO tensor with element::string element type or u8 legacy packed
 // representation, to a decompose tensor representation that may potentially
 // consist of multiple tensors. The destination format is defined by `mode` attribute.
-class StringTensorUnpack : public ov::op::Op {
+class StringTensorUnpack : public ov::op::v15::StringTensorUnpack {
```

Comment:
This is pushed accidentally. I was ensuring that this breaks old IRs. Revert it back

Reply:
reverted